docker-compose.yml

```diff
@@ -9,21 +9,32 @@ services:
       - weblate-data:/app/data
     env_file:
       - ./environment
+    ports:
+      - "8000:8000"
     restart: always
     depends_on:
       - database
       - cache
   web:
-    build: nginx
-    links:
-      - weblate
+    image: jwilder/nginx-proxy
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+      - ./certs:/etc/nginx/certs:ro
+      - /etc/nginx/vhost.d
+      - /usr/share/nginx/html
     ports:
       - "80:80"
-    volumes_from:
-      - weblate:ro
```

Comment:
I understand your point regarding performance, good catch. I'm ready to research and find a way to do the same thing without sacrificing performances. 

However if you're not willing to support HTTPs by default for docker containers, it's probably not worth the effort. I'm inclined to think this is a good thing. I would even go as far as saying it is necessary, as explained by the EFF https://www.eff.org/encrypt-the-web. In my own words I think having https deployed as the default makes a significant difference because most server installation stick with the default. Note that I'm not pushing for this for selfish reasons: the weblate setup that is installed for SecureDrop is now using HTTPs. I'm advocating for other weblate users and an increasingly encrypted web.

It is ultimately your decision and I will, of course, accept it as final.